File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-4.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        heapq.heapify(nums)
        k = len(nums) - k + 1
        while k > 0 :
            ans = heapq.heappop(nums)
            k -= 1
        print(ans)
        return (ans)

File: Data Structures & Algorithms/two-integer-sum/submission-4.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> ans ;
        unordered_map<int,int> complement;
        int n = nums.size();
        for(int i = 0 ; i < n ; i++){
            int c = target - nums[i];
            
            if(complement.count(nums[i])){
                cout<<"here";
               ans.push_back(complement[nums[i]]);
               ans.push_back(i);
               return ans;
            }
            complement[c] = i;
        }
        return ans;
    }
};
